pkg/events/core.go

```diff
@@ -262,7 +263,7 @@ var CoreEvents = map[ID]Definition{
 		params: []trace.ArgMeta{
 			{Type: "const char*", Name: "pathname"},
 			{Type: "int", Name: "flags"},
-			{Type: "mode_t", Name: "mode"},
+			{Type: "umode_t", Name: "mode"},
```

Comment:
@NDStrahilevitz we need standardise names or migrate entirely to types instead of strings. Seeing this change and double checking https://github.com/aquasecurity/tracee/commit/4737625352041787888fb44c5fbb8e1484c8dbf4#diff-561afb519199f76436d77252de43fdbc7020a8603bb0dd8226d8eea314c4ba32R29-R154, I realized that we can't be confident of the type size based on the name only, check these:

https://elixir.bootlin.com/linux/v6.11.1/source/include/linux/types.h#L23
https://elixir.bootlin.com/linux/v6.11.1/source/include/linux/types.h#L24

https://elixir.bootlin.com/linux/v6.11.1/source/arch/x86/include/uapi/asm/posix_types_32.h#L11
https://elixir.bootlin.com/linux/v6.11.1/source/include/uapi/asm-generic/posix_types.h#L24

`unsigned short` OR `unsigned int`.

I'll review all `ArgZeroValueFromType()` named types again and chose the larger option when necessary.